### DISCUSSION TITLE
Roller over previous month income, how to manage?

### DISCUSSION THREAD

**Discussion by Solverz-0** [firefly-iii/firefly-iii#6637] [Firefly III] [answered]
Lets say my salary is at the end of the month but I use this salary for all of the expenses for the following month, how is this managed in Firefly?
As the Balance block will always show a negative number until end of the month which is not the most helpful for me personally.

I think it would be good to have a Rollover Balance block to show the previous month salary income minus following months expenses in the dashboard.

Additionally, I see you have to set the total budget available manually each month but I am not sure how to cleanly use this as lets say my income salary is 1000 and I set the available budget to 1000, but as not everything has a budget this value will not be true as it will go down as I have expenses outside of budgets. Maybe if the the available budget could be configurable so you can choose different incomes (including previous month rollover income) to be part of the available budget each month and as you have expenses outside budgets each month, this value decreases?

**Comment by JC5** [comment, marked as answer]
>  As the Balance block will always show a negative number until end of the month which is not the most helpful for me personally.

The block will disappear in future versions of Firefly III.
